### PR TITLE
route steps design

### PR DESCRIPTION
**Pull request by G-Ray**
## Description
Update route steps according to new design :
- Updated fonts
- Updated layout : distance is now displayed under the direction step.
- Updated origin and destination icons, for now I used the new `svg` icon to be consistent with the new direction form. Integrating new icons asked by design + colors logic (depending on the type of poi) will be implemented in another changes set.
- Used Divider where possible and cleaned some css styles. Hopefully nothing is broken.

## Screenshots
mobile
desktop

### CODE REVIEW

File: src/scss/includes/direction.scss [hunk 1]
```diff
@@ -50,16 +50,18 @@
   overflow: auto;
 }
 
-.itinerary_leg {
-  position: relative;
-
+.itemList-item {
```
Comment (bbecquet) [inline src/scss/includes/direction.scss:53]: Make sure to scope that as you are modifying the global `.itemList-item` component here. Or if it's the target look and consistent with other uses in the app, we should modify it in its own SCSS.

**Review by bbecquet — COMMENTED** [review]
Nice simplification.
Some specific rules are still need to make sure the `RoadMapStep` component looks good even when used in the mobile step-by-step context, as now the paddings look off.
Capture d’écran de 2020-10-19 15-50-06

**Comment by G-Ray** [comment]
> Nice simplification.
> Some specific rules are still need to make sure the `RoadMapStep` component looks good even when used in the mobile step-by-step context, as now the paddings look off.
> Capture d’écran de 2020-10-19 15-50-06

Thanks, should be better now.
image

File: src/scss/includes/direction.scss (hunk 1 above)
Comment (G-Ray) [reply src/scss/includes/direction.scss:53]: thanks, this should be a temporary fix, the delimiter has disappeared a few commits ago it seems, and it should be removed in a next commit due to design changes

**Review by bbecquet — APPROVED** [review]

**Pull request merged**